ravenframework/CodeInterfaceClasses/Abce/AbceCodeInterface.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on 2023-Jul-12
+This is a CodeInterface for the ABCE code.
+"""
+
+import os
+import re
+import warnings
+import pandas as pd
+import sqlite3
+
+from ravenframework.CodeInterfaceBaseClass import CodeInterfaceBase
+from ..Generic.GenericCodeInterface import GenericParser
+
+# For Memoization testing (only benefitial if same args are sometimes passed to finalizeCodeOutput)
```

Comment:
Hm, if we want to do Memoization, it probably needs to be under user control, and should probably be done at a higher level than in an individual CodeInterface.